cmd/ebrelayer/txs/relay.go

```diff
@@ -50,5 +52,20 @@ func RelayEvent(chainId string, cdc *amino.Codec, claim *types.EthBridgeClaim) e
 
 	cliCtx.PrintResponse = true
 
-	return utils.CompleteAndBroadcastTxCLI(txBldr, cliCtx, []sdk.Msg{msg})
```

Comment:
Deconstruct the `CompleteAndBroadcastTXCLI()` call so we can automatically apply the validator's passphrase to the tx makes sense to me.